pkg/client/grpcauthorizer/kube_authorizer.go

```diff
@@ -26,7 +26,7 @@ var _ GRPCAuthorizer = &KubeGRPCAuthorizer{}
 
 // TokenReview validates the given token and returns the user and groups associated with it.
 func (k *KubeGRPCAuthorizer) TokenReview(ctx context.Context, token string) (user string, groups []string, err error) {
-	klog.V(4).Infof("TokenReview: token=%s", token)
+	klog.V(10).Infof("TokenReview: token=%s", token)
```

Comment:
nit: maybe we should delete this for security concern.

Reply:
removed